Makefile

```diff
@@ -64,12 +64,12 @@ install: go-version-check
 
 .PHONY: clean
 clean:
-	$(GO) clean -i
+	go clean -i
```

Comment:
This is not working when the project is outside GOPATH/src. Suggest to use `$(GO) clean -i ./...`, then it will work in both places.

Reply:
💯